File: AtCoder/abc202/B.swift
let s = readLine()!
var ans = ""
s.reversed().forEach {
	if $0 == "6" {
		ans += "9"
	} else if $0 == "9" {
		ans += "6"
	} else {
		ans += String($0)
	}
}
print(ans)

File: AtCoder/abc202/B.kt
private fun readLn() = readLine()!!

fun main(args: Array<String>) {
    val s = readLn()
    s.reversed().forEach {
        when (it) {
            '6' -> print(9)
            '9' -> print(6)
            else -> print(it)
        }
    }
}